src/modules/data/blocks/datetime/reducer.js

```diff
@@ -27,9 +27,11 @@ export const defaultEndMoment = moment().startOf( 'day' ).seconds( defaultEndTim
 
 const defaultStartDateTime = momentUtil.toDateTime( defaultStartMoment );
 const defaultEndDateTime = momentUtil.toDateTime( defaultEndMoment );
+const queryStartDate = globals.postObjects().tribe_events.tribe_start_date;
+const queryStartDateTime = queryStartDate + ' ' + defaultStartTime;
 
 export const DEFAULT_STATE = {
-	start: defaultStartDateTime,
+	start: queryStartDate ? queryStartDateTime : defaultStartDateTime,
```

Comment:
set line 25 above:

```
const queryStartDate = globals.postObjects().tribe_events.tribe_start_date;
export const defaultStartMoment = queryStartDate ? moment( queryStartDate ).seconds( defaultStartTimeSeconds ) : moment().startOf( 'day' ).seconds( defaultStartTimeSeconds );
```

Remove the rest of the additions here, the `defaultStartMoment` creates the `defaultStartDateTime`, which sets the `DEFAULT_STATE`.